### ISSUE TITLE
Localhost Remote hosts showing up expanded.

### ISSUE THREAD

**Issue by Yamanipanuchi** [RainbowMiner/RainbowMiner#1151]
The remote host tab shows all devices expended by default.

![image](https://user-images.githubusercontent.com/4724631/91732429-13821700-eb5d-11ea-8227-6f858baaaaab.png)

**Comment by RainbowMiner** [comment]
Yes, that was part of my past update. Do you prefer non-expanded? If yes, I'll have to add a parameter.

**Comment by Yamanipanuchi** [comment]
I would prefer to see most of the shown information all the time. But in a more condensed format would be more effective. I wonder if the expand option could be eliminated and rows could be combined. Or put a different way, I usually glance weather something is up or down, But I always go through each rig and see what it is mining also. If one screen could show the rig status (not expanded) and the mining details (Expanded) in one line to keep the screen compact.

I have 6 rigs I monitor using the server option, And sometimes more. It takes up a lot of space. I mentioned a while back about potentially have a dashboard for server systems that display the information differently. Maybe? 

https://github.com/RainbowMiner/RainbowMiner/issues/646

**Comment by Yamanipanuchi** [comment]
An alternative idea if the previous idea doesn't work, Is to combine the two lines into one? In the case where a rig has more than one active miner have two lines with the same device name?

I've attempted to dig into code for the design of the web interface myself. Unfortunately it is just a bit over my capabilities.

![image](https://user-images.githubusercontent.com/4724631/91863463-ff551d00-ec23-11ea-9a82-77861f26debd.png)

**Comment by RainbowMiner** [comment]
The web UI was rewritten on 2026-07-15 (d4c520a7) and now uses Tabulator 6.3.1 instead of the old table code, so the expand/collapse behaviour described here no longer exists in this form. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
